.github/workflows/ci.yml

```
@@ -136,3 +136,13 @@ jobs:
       # note: using "@main" because "@${{env.BUILDER_VERSION}}" doesn't work
       # https://github.com/actions/runner/issues/480
       uses: awslabs/aws-crt-builder/.github/actions/check-submodules@main
+
+  check-lockfile-version:
+    runs-on: ubuntu-20.04 # latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Check for edits to package-lock.json
+        run: |
+          ./utils/lockfile_version_validation.py
```

I know I've advocated against shell scripts in the past, but this is a pretty simple one-liner
```suggestion
          test `jq -r '.lockfileVersion' package-lock.json` = 2
```

---

That's super elegant!!

---

Wait, I thought we are gona go with version 1?

---

oops. yeah 1